### DISCUSSION TITLE
Make it easier to overwrite traefik values

### DISCUSSION THREAD

**Discussion by schlichtanders** [mysticaltech/terraform-hcloud-kube-hetzner#1418] [Ideas]
currently one has to be super careful to really take all the traefik value hetzner provides by default into ones own traefik_values variable.

It would be awesome if these could be merged more granulary so that for instance adding some tolerations does no look like a technical rabbit hole.

**Comment by mysticaltech** [comment]
Hum, as a rule of thumb, the best way is to pull the helm related crd values from the cluster.

kubectl get crds | grep helm

Pull each of those, especially the helm config, and then you get the right values to change. There's no other practical way to do it, but it's really just a few commands to run, especially with AI now, with something like claude code, it's done in 2 minutes.
